be1-go/channel/election/verification_test.go

```diff
@@ -102,6 +102,26 @@ func TestVerify_ElectionOpen_already_closed(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestVerify_ElectionOpen_created_time_less_than_create_time_setup(t *testing.T) {
+	// create the opened election channel with election open time less than election creation time
```

Comment:
Comments must be formatted at 80 chars:
```suggestion
  // create the opened election channel with election open time less than
  // election creation time
```